templates/notices/error.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Show error messages
+ *
+ * This template can be overridden by copying it to yourtheme/woocommerce/notices/error.php.
+ *
+ * HOWEVER, on occasion WooCommerce will need to update template files and you
+ * (the theme developer) will need to copy the new files to your theme to
+ * maintain compatibility. We try to do this as little as possible, but it does
+ * happen. When this occurs the version of the template file will be bumped and
+ * the readme will list any important changes.
+ *
+ * @see     https://docs.woocommerce.com/document/template-structure/
+ * @package WooCommerce\Templates
+ * @version 3.9.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! $notices ) {
+	return;
+}
+
+$notices[] = $notices[0];
+
+$multiple = count( $notices ) > 1;
+
+if ( ! $multiple ) {
```

Comment:
Can't we avoid repeating html and that long svg by moving this condition inside the .wc-block-components-notice-banner__content wrapper? While adding a ternary for the inconvenient wc_get_notice_data_attr() on the main wrapper..